sql-plugin/src/main/scala/org/apache/spark/sql/rapids/arithmetic.scala

```diff
@@ -282,6 +282,46 @@ object GpuDivModLike {
       case t => throw new IllegalArgumentException(s"Unexpected type: $t")
     }
   }
+
+  def isDivOverflow(left: GpuColumnVector, right: GpuColumnVector): Boolean = {
```

Comment:
nit: A comment here might be nice explaining the exact condition that this can happen in.

Reply:
Thanks for reivew. Added.